src/pages/resultsView/download/DownloadUtils.ts

```diff
@@ -22,20 +21,54 @@ export interface IDownloadFileRow {
 }
 
 export function generateOqlData(datum: GeneticTrackDatum,
```

Comment:
we may want to add more unit tests for this function to cover the latest changes